lib/iris/experimental/um.py

```diff
@@ -372,6 +449,18 @@ def _with_source(self):
             if close_required:
                 self.source.close()
 
+    def _read_raw_payload_bytes(self):
+        # Return the raw data payload, as an array of bytes.
+        # This is independent of the packing representation.
+        field = self.lookup_entry
+        with self._with_source():
+            self.source.seek(self.offset)
+            data_size = ((field.lbnrec * 2) - 1) * _WGDOS_SIZE
```

Comment:
The WGDOS_SIZE usage here is concerning. Shouldn'd this be dependent on lbpack and word size?